tests/decode_images.rs

```diff
@@ -391,6 +391,10 @@ fn oom() {
 
     match err {
         tiff::TiffError::LimitsExceeded => {}
+        tiff::TiffError::UnsupportedError(tiff::TiffUnsupportedError::InterpretationWithBits(
+            _,
+            _,
+        )) => {}
```

Comment:
nit: this can be shortened to `tiff::TiffError::UnsupportedError(tiff::TiffUnsupportedError::InterpretationWithBits(..)) => {}`